README.md

```diff
@@ -47,6 +47,19 @@ This README.md files continues by explaining you how to compile MeasurementKit
 on a Unix-like platform (e.g. Linux, MacOS), on Android, and on iOS. Finally it
 provides an example of usage of MeasurementKit.
 
+## Index 
```

Comment:
Rather than writing `## Index` I'd modify the previous sentence and say something like that the following index illustrates the content of the remainder of this file.